bigquery_etl/cli/dryrun.py

```diff
@@ -134,5 +141,11 @@ def _sql_file_valid(
         respect_skip=respect_skip,
     )
     if validate_schemas:
-        return result.validate_schema()
-    return result.is_valid()
+        try:
+            success = result.validate_schema()
+        except Exception as e:  # validate_schema raises base exception
+            click.echo(e, err=True)
```

Comment:
I'd suggest adding an error message (including the file in question) with the exception at the end.  Otherwise it might be hard to tell what the exception is for in the deluge of parallel-processed dry-runs.

Reply:
The exceptions here always have the file name like `sql/moz-fx-data-shared-prod/search_derived/acer_cohort_v1/query.sql ERROR` so I didn't add it but I guess that can change

Reply:
OK, never mind then. 👍